Library/Formula/percona-toolkit.rb

```diff
@@ -15,15 +16,15 @@ class PerconaToolkit < Formula
   depends_on "openssl"
 
   resource "DBD::mysql" do
-    url "http://search.cpan.org/CPAN/authors/id/C/CA/CAPTTOFU/DBD-mysql-4.027.tar.gz"
-    mirror "http://search.mcpan.org/CPAN/authors/id/C/CA/CAPTTOFU/DBD-mysql-4.027.tar.gz"
-    sha1 "3bf1edd6f0b4f6144b2aaa715c80df3fb1cd2119"
+    url "http://www.cpan.org/authors/id/C/CA/CAPTTOFU/DBD-mysql-4.032_01.tar.gz"
+    mirror "http://search.mcpan.org/CPAN/authors/id/C/CA/CAPTTOFU/DBD-mysql-4.032_01.tar.gz"
```

Comment:
I suggest this new SSL/TLS URL for `mirror` (above one redirects to it):
https://cpan.metacpan.org/authors/id/C/CA/CAPTTOFU/DBD-mysql-4.032_01.tar.gz